db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180221142221) do
+ActiveRecord::Schema.define(version: 20180312112638) do
```

Comment:
you'll need to rebase the [patch](https://github.com/kubic-project/velum/blob/master/packaging/suse/patches/0_set_default_salt_events_alter_time_column_value.rpm.patch) otherwise it wont build (due to the strictness in our configuration)

Reply:
THE patch, right. I **always** forget.